public/sandbox-data/apprenticeship-levy/employers/191GPRPA6NZJJ/employed/AA678901C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "191GPRPA6NZJJ",
+    "nino": "AA678901C",
```

Comment:
Use of Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.